spec/api.json

```diff
@@ -4818,6 +4818,141 @@
           ]
         }
       },
+      "/organizations/{orgId}/services": {
+        "get": {
+          "summary": "list services for an organization",
+          "tags": [
+            "Organizations"
+          ],
+          "operationId": "getServicesForOrg",
```

Comment:
This is good. I was looking for consistency and noticed the `getProjectsByOrId` should be `getProjectsForOrg`. Not related to your changes and not a serious requirement though.

Reply:
ok added a change